Game/Assets/AdamitoPleaseWork.fbx.meta

```diff
@@ -1,15 +1,15 @@
 {
     "Resources": [
         {
-            "Id": 17564419156017004516,
+            "Id": 5938164575651497437,
             "Type": "Material"
         },
         {
-            "Id": 10141510311928753299,
+            "Id": 13649703379845438305,
             "Type": "Mesh"
         },
         {
-            "Id": 6064428306325331534,
+            "Id": 1876992495970482398,
```

Comment:
The IDs on meta files should never change. Revert this file's changes. This could break the scenes.